src/clients/QBittorrent.ts

```diff
@@ -244,39 +240,53 @@ export default class QBittorrent implements TorrentClient {
 	 */
 	async generateCorrectSavePath(
 		searchee: Searchee,
-		{ save_path },
-	): Promise<{ save_path: string }> {
-		const subfolderContentLayout =
-			await this.isSubfolderContentLayout(searchee);
+		torrentInfo: TorrentInfo,
+	): Promise<string> {
+		const subfolderContentLayout = await this.isSubfolderContentLayout(
+			searchee,
+			torrentInfo,
+		);
 		if (subfolderContentLayout) {
-			return {
-				save_path: join(save_path, stripExtension(searchee.name)),
-			};
+			return dirname(torrentInfo.content_path);
```

Comment:
When i was doing research on the subfolder content type, I read that if you are using a incomplete folder, content path will be the temp folder prior to the files reaching their final destination - which is the save_path property- so I'm curious what the result is if the files are moving (or pre finished moving?) and whether that is considered "complete" but the files are moving and what the content_path actually would be in transit?

Reply:
Possibly irrelevant depending on when the actual completion callback is made from libtorrent, just a thought I had so i figured I'd comment to discuss since this changes things if not handled the way we are assuming.

Reply:
We check `progress` which I'm unsure of it's value considering the torrent's percent complete would be 100% `( progress === 1)` despite the "downloading/moving" operation being incomplete. There might need to be further checks done to validate isComplete given the operations taking place in this manner.

Reply:
We could also check the status for moving/rechecking or any other scenario as a part of that completeness check. Or just require it to be `uploading/pausedUP/queuedUP/stalledUP/forcedUP`